src/QandaAtom.tsx

```diff
@@ -1,7 +1,450 @@
+    border: 0px;
+`;
+
+const plusStyling = css`
+    margin-right: 12px;
+    margin-bottom: 6px;
+    width: 33px !important;
+    fill: white;
+    height: 28px !important;
+`;
+
+const minusStyling = css`
+    margin-right: 14px;
+    margin-bottom: 6px;
+    width: 30px !important;
+    fill: white;
+    height: 25px !important;
+    padding-left: 4px;
+`;
+
+const iconSpacing = css`
+    display: inline-flex;
+    align-items: center;
+    ${textSans.small()};
+`;
+
+const thumbIcon = css`
+    width: 16px;
+    height: 16px;
+`;
+
+const footerStyling = css`
+    font-size: 13px;
+    line-height: 16px;
+    display: flex;
+    justify-content: flex-end;
+`;
+
+const footerSnippet = css`
+    display: flex;
+    align-items: center;
+    ${textSans.xsmall()};
+`;
+
+const footerFeedback = css`
+    ${textSans.xsmall()};
+    height: 28px;
+`;
+
+const creditStyling = css`
+    ${textSans.xsmall()};
+    margin: 12px 0;
+`;
+
+const creditIconStyling = css`
+    height: 12px;
+    overflow: visible;
+    width: 6px;
+    fill: white;
+`;
+
+const creditSpanStyling = css`
+    display: inline-flex;
+    background: #bdbdbd;
+    border-radius: 100%;
+    width: 16px;
+    height: 16px;
+    align-items: center;
+    justify-content: center;
+    margin-right: 5px;
+`;
+
+// Components
+const Container = ({
+    id,
+    title,
+    children,
+    expandHandler,
+}: {
+    id: string;
+    title: string;
+    children: React.ReactNode;
+    expandHandler: () => void;
+}) => {
+    return (
+        <div
+            className={containerStyling}
+            data-atom-id={id}
+            data-atom-type="qanda"
+        >
+            <details
+                data-snippet-id={id}
+                data-snippet-type="qanda"
+                className={detailStyling}
+            >
+                <Summary title={title} expandHandler={expandHandler}></Summary>
+                {children}
+            </details>
+        </div>
+    );
+};
+
+const Summary = ({
+    title,
+    expandHandler,
+}: {
+    title: string;
+    expandHandler: () => void;
+}) => {
+    const [hasBeenExpanded, setExpandStatus] = useState(false);
+    const [expandEventSent, setExpandEventFired] = useState(false);
+    return (
+        <summary className={summaryStyling}>
+            <span
+                className={css`
+                    color: #e00000;
+                    display: block;
+                    ${body.medium({
+                        lineHeight: 'tight',
+                        fontWeight: 'bold',
+                    })};
+                `}
+            >
+                Q&amp;A
+            </span>
+            <h4
+                className={css`
+                    ${headline.xxxsmall({
+                        fontWeight: 'medium',
+                    })};
+                    margin: 0;
+                    line-height: 22px;
+                `}
+            >
+                {title}
+            </h4>
+            <span
```

Comment:
Worth double check the keyboard navigation, I assume the 'details' element allows tabbing / spacebar to open, but as we're aria-hidden on this we should make sure it is still accessible to screen reader /keyboard users